ee/tabby-ui/components/message-markdown/index.tsx

```diff
@@ -170,6 +171,15 @@ export function MessageMarkdown({
       }
     }
 
+    // Add think placeholders to the matches.
+    findMatches(
+      MARKDOWN_THINK_REGEX,
+      ThinkPlaceholder,
+      (match: RegExpExecArray) => ({
+        content: decodeURIComponent(match[1])
+      })
+    )
```

Comment:
instead of converting from PLACEHOLDER_THINK_REGEX to MARKDOWN_THINK_REGEX, can we just use PLACEHOLDER_THINK_REGEX here to render the think content?